src/main/resources/styles/reporting.css

```diff
@@ -191,43 +163,49 @@ table.stats-table td {
 	background-color: white;
 }
 
-table.stats-table .header {
+table.stats-table tr.header {
 	background-color: #66CCEE;
 }
 
-table.stats-table .summary {
+table.stats-table tr.summary {
 	background-color: lightgray;
 }
 
-table.data-table {
-	color: black;
-	border-width: 1px;
-	border-spacing: 2px;
-	border-style: outset;
-	border-color: gray;
-	border-collapse: collapse;
-	background-color: beige;
+table.stats-table td.duration {
+	text-align: right;
+	white-space: nowrap;
 }
 
-table.data-table th {
-	color: black;
-	border-width: 1px;
-	padding: 5px;
-	border-style: inset;
-	border-color: gray;
-	background-color: #66CCEE;
+table.stats-table td.tagname {
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Element (td.tagname) is overqualified, just use .tagname without element name.](https://www.codacy.com/app/damianszczepanik/cucumber-reporting/file/1993444831/issues/source?bid=3059308&fileBranchId=3120225#l179)